### DISCUSSION TITLE
Does DCjs Support Broken Axis/Scale Break In Bar Charts

### DISCUSSION THREAD

**Discussion by lumubashaka** [dc-js/dc.js#1804] [Q&A]
Does DCjs support broken axis/scale break in bar charts, like some of the other charting libraries.

**Comment by gordonwoodhull** [comment]
I’m not sure! In principle any [D3 scale](https://github.com/d3/d3-scale/blob/v3.2.2/README.md) should work, although some may need troubleshooting or workarounds.

It sounds like this would be a discontinuous scale... thought I saw support for that in D3 but not finding that now.
